cloud/pkg/cloudhub/handler/messagehandler.go

```diff
@@ -250,10 +255,9 @@ func (mh *MessageHandle) ServeConn(hi hubio.CloudHubIO, info *model.HubInfo) {
 
 	klog.Infof("edge node %s for project %s connected", info.NodeID, info.ProjectID)
 	exitServe := make(chan ExitCode, 3)
-	stopSendMsg := make(chan struct{})
 
 	for _, handle := range mh.Handlers {
-		go handle(hi, info, exitServe, stopSendMsg)
+		go handle(info, exitServe)
 	}
 
 	code := <-exitServe
```

Comment:
ExitCode there seem only could be `nodeStop`, which is generated by `kubectl delete node`

Reply:
Yes, now if the node haven't been deleted, we will not quit the goutines